packages/frontend2/src/app/_components/callout.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { type ReactNode } from 'react'
+import { cn } from '~/utils/cn'
+
+export interface CalloutProps {
+  color?: 'red' | 'yellow' | 'blue' | 'gray'
+  icon: ReactNode
+  body: ReactNode
+  message?: ReactNode
+  className?: string
+  hoverable?: boolean
+  small?: boolean
+}
+
+export function Callout({
+  color,
+  icon,
+  body,
+  message,
+  className,
+  hoverable,
+  small,
+}: CalloutProps) {
+  let background: string
+  if (hoverable && color === 'red') {
+    background = 'bg-red-600/20 hover:bg-red-500/20'
+  } else if (hoverable && color === 'yellow') {
+    background = 'bg-yellow-700/20 hover:bg-opacity-40/20'
+  } else if (color === 'red') {
+    background = 'bg-red-600/20'
+  } else if (color === 'yellow') {
+    background = 'bg-yellow-700/20'
+  } else if (color === 'blue') {
+    background = 'bg-blue-700/20'
+  } else if (color === 'gray') {
+    background = 'bg-gray-500/20'
+  } else {
+    background = ''
+  }
```

Comment:
Can we use variants here? I saw you were doing a refactor like this somewhere else.